packages/components/src/suggestions/test/index.js

```diff
@@ -23,32 +27,38 @@ describe( '<Suggestions>', () => {
 	} );
 
 	test( 'render basic list of suggestions', () => {
-		const wrapper = shallow( <Suggestions { ...defaultProps } /> );
-		expect( wrapper.find( Item ) ).toHaveLength( 3 );
-		expect( wrapper ).toMatchSnapshot();
+		const { container } = render( <Suggestions { ...defaultProps } /> );
+
+		expect( screen.getAllByRole( 'button' ) ).toHaveLength( 3 );
+
+		expect( container.firstChild ).toMatchSnapshot();
 	} );
 
 	test( 'highlights the query inside the suggestions', () => {
-		const wrapper = shallow( <Suggestions { ...defaultProps } query="LE" /> );
-		expect( wrapper.find( { label: 'Pear' } ).prop( 'hasHighlight' ) ).toBe( false );
-		expect( wrapper.find( { label: 'Orange' } ).prop( 'hasHighlight' ) ).toBe( false );
-		expect( wrapper.find( { label: 'Apple' } ).prop( 'hasHighlight' ) ).toBe( true );
-		expect( wrapper.find( { label: 'Apple' } ).prop( 'query' ) ).toBe( 'LE' );
+		render( <Suggestions { ...defaultProps } query="LE" /> );
```

Comment:
I feel like it could still be useful to verify that the suggestions are arranged in order of the query. E.g. if the input list is

```js
[
  { label: 'foo' },
  { label: 'bar' },
  { label: 'baz' },
]
```

and the query is "az", then we could test that the first suggestion is "baz" and not "foo"

Reply:
Cool, now also checking for the suggestion label in https://github.com/Automattic/wp-calypso/pull/70882/commits/b63cfd01c180d8d309e6fe0d047ec08465984799.